conf/cassandra.yaml

```diff
@@ -1362,6 +1362,11 @@ server_encryption_options:
   # Set to a valid keystore if internode_encryption is dc, rack or all
   keystore: conf/.keystore
   keystore_password: cassandra
```

Comment:
Looking at this one last time, don't we want to comment out all 4 of the `keystore_password` and `truststore_password` field in the YAML? If we leave them there, and operators just start using the YAML we ship, we will essentially have preserved the "cassandra" default. The point of all this is that we want to force the operator to choose passwords, right?

CC @smiklosovic

Reply:
I agree with you @maedhroz. Will wait to hear Stefan's comment and would make appropriate changes.

Reply:
@smiklosovic any comments before we can close this?

Reply:
Aha, this is great catch. Yes we should not have them explicitly written there. Just comment it out and leave "cassandra" as password there to have some hint what to put there so people have easy job. "cassandra" password is used sometimes during testing etc, some testing clusters just have "cassandra" password for their certs / keys, same as they have "cassandra" password by default for CQL when using PasswordAuthenticator.

Reply:
That works for me.

Reply:
Cool. So @smiklosovic and @maedhroz I have [a separate PR](https://github.com/apache/cassandra/pull/2273) for the 4.1 backport and in this PR I can comment out the password configuration from the cassandra yaml since this will be target for 5.0 release. Please let me know if that understanding is correct.

Reply:
Done commenting out the passwords in the cassandra yaml in this PR.

Reply:
thank you, I will take a look shortly. Thanks for the patience.

Reply:
@maulin-vasavada looks good, please apply what was discussed here in 4.1 branch and we can finally ship it.

Reply:
Cool thanks @smiklosovic I've the backported changes for 4.1 [here in a different PR](https://github.com/apache/cassandra/pull/2273). Will remove news.txt update from there. Thanks for helping.